tests/test_django_orm.py

```diff
@@ -58,10 +59,39 @@ def test_field_unpickled(self):
         self.assertTrue(isinstance(self.field.to_python(self.pickle),
                                    Credentials))
 
+    def test_field_unpickled_none(self):
+        self.assertEqual(self.field.to_python(None), None)
+
     def test_field_pickled(self):
         prep_value = self.field.get_db_prep_value(self.credentials,
                                                   connection=None)
-        self.assertEqual(prep_value, self.pickle)
+        self.assertEqual(_to_bytes(prep_value), self.pickle)
+
+
+class TestCredentialsFieldViaModel(unittest.TestCase):
+
+    class TestModel(models.Model):
+        credentials = CredentialsField()
+
+    def setUp(self):
+        self.model = self.TestModel()
+        # using the meta api:
+        # https://docs.djangoproject.com/en/1.8/ref/models/meta/#field-access-api
```

Comment:
the `get_field` method (and the `_meta` API) has existed for a long time, but wasn't documented until now. Not sure when it was added, but it definitely can be found in Django `1.3.*`, which is pretty old version (more than 2 years old).